### PR TITLE
MPI-Operator: Add docs for the schedulingPolicy

### PR DESCRIPTION
**Pull request by tenzen-y**
Since https://github.com/kubeflow/mpi-operator/pull/520, we introduce the schedulingPolicy to mpi-operator.
So I added docs for that.

/assign @alculquicondor 
/hold for https://github.com/kubeflow/mpi-operator/pull/520.

### CODE REVIEW

**Comment by alculquicondor** [comment]
Why is this mentioning the training operator?

File: content/en/docs/components/training/job-scheduling.md
Comment (tenzen-y) [inline content/en/docs/components/training/job-scheduling.md:78]: > Why is this mentioning the training operator?

@alculquicondor Does that mean this line?

The `job-scheduling.md` is not docs only for the mpi-operator. However, we have not had docs on how to set up gang-scheduling in the mpi-operator and only have the docs for the training-operator.

So I added new docs for the mpi-operator since the training-operator and the mpi-operator have different ways of setting up the gang-scheduling.

Comment (alculquicondor) [reply content/en/docs/components/training/job-scheduling.md:78]: ahh, I missed `mpi-operator` below.

File: content/en/docs/components/training/job-scheduling.md
Comment (alculquicondor) [inline content/en/docs/components/training/job-scheduling.md:100]: This should also work, right?

```suggestion
+       - --gang-scheduling=volcano
```

File: content/en/docs/components/training/job-scheduling.md
Comment (alculquicondor) [inline content/en/docs/components/training/job-scheduling.md:104]: I think it's better not to put a tag so that it targets latest, or put a released tag.

**Review by alculquicondor — COMMENTED** [review]
/hold

We should probably only merge this after we have a release

**Comment by tenzen-y** [comment]
> We should probably only merge this after we have a release

It makes sense. Let's merge this PR once we release v0.4.0.

File: content/en/docs/components/training/job-scheduling.md
Comment (tenzen-y) [reply content/en/docs/components/training/job-scheduling.md:100]: Yes, you're right. 
I used a space delimiter, not an equal delimiter since we use a space delimiter in L102-L103.

File: content/en/docs/components/training/job-scheduling.md
Comment (tenzen-y) [reply content/en/docs/components/training/job-scheduling.md:104]: Good point. Let's use the `v0.4.0`. I will fix the tag once we release the mpi-operator v0.4.0.

File: content/en/docs/components/training/job-scheduling.md
Comment (tenzen-y) [reply content/en/docs/components/training/job-scheduling.md:100]: Done.

File: content/en/docs/components/training/job-scheduling.md
Comment (alculquicondor) [reply content/en/docs/components/training/job-scheduling.md:104]: 0.4.0

Comment (tenzen-y) [reply content/en/docs/components/training/job-scheduling.md:104]: Done.

Comment (alculquicondor) [reply content/en/docs/components/training/job-scheduling.md:104]: or just `0.4`

Comment (tenzen-y) [reply content/en/docs/components/training/job-scheduling.md]: selected `0.4.0`.

File: content/en/docs/components/training/mpi.md
Comment (alculquicondor) [inline content/en/docs/components/training/mpi.md:102]: Also indicate that if you don't set the fields, mpi-operator populates them based on .....

Comment (tenzen-y) [reply content/en/docs/components/training/mpi.md:102]: agree

**Review by alculquicondor — COMMENTED** [review]
/lgtm
/approve

**Review by terrytangyuan — APPROVED** [review]
/lgtm

**Pull request merged**
